web/admin/application/library/IvozProvider/Klear/Ghost/SchedulerSuccess.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+use Ivoz\Core\Application\Service\DataGateway;
+use Ivoz\Kam\Domain\Model\UsersLocation\UsersLocation;
+use Ivoz\Kam\Domain\Model\UsersLocation\UsersLocationDto;
+use Ivoz\Provider\Domain\Model\Domain\Domain;
+use Ivoz\Provider\Domain\Model\Domain\DomainDto;
+use Ivoz\Provider\Domain\Model\Friend\FriendDto;
+use Ivoz\Provider\Domain\Model\ResidentialDevice\ResidentialDeviceDto;
+use Ivoz\Provider\Domain\Model\Terminal\TerminalDto;
+
+class IvozProvider_Klear_Ghost_SchedulerSuccess extends KlearMatrix_Model_Field_Ghost_Abstract
+{
+
+    /** @var \Ivoz\Provider\Domain\Model\CallCsvScheduler\CallCsvSchedulerDto $model */
+    public function getCallCsvSchedulerIcon($model)
+    {
+        $errorMsg = $model->getLastExecutionError();
+        $lastExecution = $model->getLastExecution();
+        $lastExecutionStr = $lastExecution
+            ? $lastExecution->format('Y-m-d H:i:s')
+            : '';
+
```

Comment:
You could return empty string response if $lastExecutionStr is empty (for the first time running the scheduler)